tutorials/hyperspectral/Work-With-Hyperspectral-Data-In-R.Rmd

```diff
@@ -211,60 +213,62 @@ that covers 800 nm-805 nm might have a FWHM of 2.5 and a wavelength value of 802
 
```

Comment:
Line 211: Add "Source: National Ecological Observatory Network (NEON)".  

Also in general, check for this, especially with these older lessons.

Reply:
Yes, good point. I have added that to this tutorial, and added a new line to my 'tutorial review checklist'